samples/expo/app/(tabs)/index.tsx

```diff
@@ -20,6 +21,12 @@ export default function TabOneScreen() {
       <Text>Update ID: {currentlyRunning.updateId}</Text>
       <Text>Channel: {currentlyRunning.channel}</Text>
       <Text>Runtime Version: {currentlyRunning.runtimeVersion}</Text>
+      <Button
+        title="Open DevMenu"
+        onPress={() => {
+          DevClient.openMenu();
```

Comment:
Q: what if we disable this button on the web? It seems to only run on mobile
![image](https://github.com/user-attachments/assets/c7d549b8-5243-41ef-beb0-95a015157ed6)

Reply:
I think we should since [the documentation](https://docs.expo.dev/versions/latest/sdk/dev-client/) does not include `Web` as a supported platform 👍

Reply:
Thank you that's a good catch.